packages/elemental/elemental/generated-changes/overlay/values.yaml.orig

```diff
@@ -0,0 +1,43 @@
+image:
```

Comment:
Was this file meant to be part of the commit? This `.orig` suffix smells a like a merging/comparing branches leftover

Reply:
good catch, removed